addon/templates/components/flexberry-ddau-slider.hbs

```diff
@@ -0,0 +1,15 @@
+{{!--
+  Binding through get-helper value=(get this "value") is a way to achieve one way binding.
+  It makes value immutable within component, what is major rule of "Data Down Actions Up (DDAU)" pattern:
+  component should display data, but not mutate it by its own.
+--}}
+{{ui-slider
```

Comment:
I won't insist to change already used 3dparty library, but see [semantic-ui-range](https://github.com/tyleryasaka/semantic-ui-range) it looks more similar to other semantic modules.